src/Simplex/Chat.hs

```diff
@@ -326,8 +338,10 @@ processChatCommand = \case
       ct@Contact {localDisplayName} <- withStore $ \st -> getContact st userId chatId
       withStore (\st -> getContactGroupNames st userId ct) >>= \case
         [] -> do
+          files <- withStore $ \st -> getContactFiles st userId ct
```

Comment:
this is wrong. You are loading files unconditionally, and then, further down the line, you check whether the files should be deleted (if folder is set). Instead you should check the folder being set here and only load files/call deletion if it is set.